docs/concepts/endpoint.md

```diff
@@ -0,0 +1,95 @@
+---
+id: endpoint
+title: Endpoint
+---
+
+Endpoints in ZIO HTTP are defined using the `Endpoint` object's combinators, which provide a type-safe way to specify various aspects of the endpoint. For instance, consider defining endpoints for retrieving user information and user posts:
+
+```scala mdoc
```

Comment:
Have you seen how this snippet will be rendered?

Reply:
yup